Agenda/Implementation/Agenda/agenda.css

```diff
@@ -217,4 +217,21 @@ nav {
 .alignRight a {
   text-decoration: none !important;
   color: white !important;
+}
```

Comment:
Are all these classes and styles actually being used somewhere? Once a design is decided upon, then all unused styles and respective class names should probably be removed to avoid confusion later on

Reply:
Yeah, these alignleft right classes are used to align the menus in mobile version

Reply:
👍